src/data/markdown/docs/02 javascript api/07 k6-experimental/01 browser.md

```diff
@@ -69,3 +97,44 @@ export default async function () {
 | [Response](/javascript-api/k6-experimental/browser/response/) <BWIPT />             | Represents the response received by the [`Page`](/javascript-api/k6-experimental/browser/page/).                                                                            |
 | [Touchscreen](/javascript-api/k6-experimental/browser/touchscreen/)                 | Used to simulate touch interactions with the associated [`Page`](/javascript-api/k6-experimental/browser/page/).                                                            |
 | [Worker](/javascript-api/k6-experimental/browser/worker/)                           | Represents a [WebWorker](https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API).                                                            |
+
+
+### Devices Example
+
+To emulate the browser behaviour on a mobile device and approximately measure the browser performance, you can import `devices` from `k6/experimental/browser`.
+
+<CodeGroup labels={["script.js"]} lineNumbers={[true]}>
+
+```javascript
+import { browser, devices } from 'k6/experimental/browser';
+
+export const options = {
+  scenarios: {
+    browser: {
+      executor: 'shared-iterations',
+      options: {
+        browser: {
+            type: 'chromium',
+        },
+      },
+    },
+  },
+  thresholds: {
+    checks: ["rate==1.0"]
+  }
+}
+
+export default async function () {
+  const iphoneX = devices['iPhone X'];
+  const context = browser.newContext();
+  const page = context.newPage();
+
+  try {
+    await page.goto('https://test.k6.io/');
+  } finally {
+    page.close();
+  }
```

Comment:
Do we call `BrowserContext.close()` implicitly upon termination? If so, do we still need to close pages at the end of the script?

Reply:
There can be multiple pages but only one browser context. So users need to close the pages. Let's add an explanation about this, too.

Reply:
Fair enough, let's keep it in mind for the upcoming changes 👍